docs/index.md

```diff
@@ -101,3 +101,13 @@ order by
 | 172.16.0.25   | GET            | /static/main.css | 304    | 5432          |
 +---------------+----------------+------------------+--------+---------------+
 ```
+
+## Detections as Code with Powerpipe
```

Comment:
Even though we don't include the new screenshot here compared to the README.md, can you please add it to the top of this doc?